ot/da.py

```diff
@@ -658,7 +658,7 @@ def __init__(self, metric='sqeuclidean'):
         self.metric = metric
         self.computed = False
 
-    def fit(self, xs, xt, ws=None, wt=None, norm=None):
+    def fit(self, xs, xt, ws=None, wt=None, norm=None, numItermax=10000):
```

Comment:
the default value for numItermax should be 100000 because it allows out of the box to handle larger matrices. Same for emd and emd2.